package.json

```diff
@@ -42,10 +42,25 @@
     "format-php": "composer format",
     "phpstan": "composer phpstan",
     "lint-php": "composer lint:all",
-    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test",
-    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite",
-    "test-php-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test:plugins",
-    "test-php-multisite-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite:plugins",
+    "test:php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer phpunit",
```

Comment:
We should update cmd related changes to handbook. Here: https://make.wordpress.org/performance/handbook/performance-lab/#useful-commands

Reply:
cc @felixarntz. 

Is there a way to update docs on `make.wordpress.org` from this repo?